zerver/openapi/zulip.yaml

```diff
@@ -4056,13 +4056,48 @@ paths:
                               example:
                                 {
                                   "id": 0,
-                                  "immediate": true,
                                   "server_generation": 1619334181,
                                   "type": "restart",
                                   "zulip_feature_level": 57,
                                   "zulip_version": "5.0-dev-1650-gc3fd37755f",
                                   "zulip_merge_base": "5.0-dev-1646-gea6b21cd8c",
                                 }
+                            - type: object
+                              description: |
+                                An event which signals to website-based clients to update the
```

Comment:
"An event which signals the official Zulip web app to update" is probably clearest.